lib/layout/dummyLayoutV2.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * dummylayoutV2.js
+ *
+ * this creates a basic poorly laid out graph
+ *
+ * @author John O'Brien.
+ */
+
+var Node = require('../model/node.js');
+var Graph = require('../model/graph.js');
+var Port = require('../model/port.js');
+
+// Create a graph instance.
+
+function buildGraph(graphs) {
```

Comment:
To clarify, you should use the syntax `var buildGraph = function(graphs) {` to keep from making the function global. Then you can export the function using `module.exports = buildGraph;`.